src/Groups/matrices/MatGrp.jl

```diff
@@ -205,12 +205,25 @@ function assign_from_description(G::MatrixGroup)
 end
 
 # return the G.sym if isdefined(G, :sym); otherwise, the field :sym is computed and set using information from other defined fields
-function Base.getproperty(G::MatrixGroup, sym::Symbol)
+function Base.getproperty(G::MatrixGroup{T}, sym::Symbol) where T
 
    isdefined(G,sym) && return getfield(G,sym)
 
    if sym === :ring_iso
-      G.ring_iso = iso_oscar_gap(G.ring)
+      if T === QQBarFieldElem
+         # get all matrix entries into one vector
+         entries = vcat([reshape(collect(matrix(g)),:) for g in gens(G)]...)
```

Comment:
```suggestion
         entries = reduce(vcat, vec(collect(matrix(g))) for g in gens(G))
```
I hope I did not miscount the `)`.

Reply:
Ah, `vec` was the function I was looking for and could not remember, thanks.